packages/@okta/vuepress-site/docs/reference/api/apps/index.md

```diff
@@ -4420,7 +4420,7 @@ YoEdncuy+GQGzE9yLOhC4HNfHQXpqp2tMPdRlw==</ds:X509Certificate>
 
 <ApiOperation method="post" url="/api/v1/apps/${applicationId}/credentials/csrs" />
 
-Generates a new key pair and returns the Certificate Signing Request for it
+Generates a new key pair and returns the CSR (Certificate Signing Request) for it. The information in a CSR is used by the Certificate Authority (CA) to verify and create your certificate. It also contains the public key that will be included in your certificate.
```

Comment:
Changes to first sentence:
The term goes first and then the abbreviation in parenthesis.
No need to include "for it". It's clear that the CSR is "for it".

Changes to last sentence:
future tense changed to present tense.

Generates a new key pair and returns the Certificate Signing Request (CSR). The information in a CSR is used by the Certificate Authority (CA) to verify and create your certificate. It also contains the public key that is included in your certificate.

Reply:
Updated.